### PR TITLE
[blazor-wasm] Ensure PDBArtifacts artifact is created

### PR DESCRIPTION
**Pull request by dougbu**
- dotnet/aspnetcore-internal#3535
- avoid build warnings when publishing symbols in post-build

### CODE REVIEW

File: eng/Publishing.props
Comment (pranavkm) [inline eng/Publishing.props]: Is this entirely empty in this branch? We aren't doing anything particular to not publish PDBs.

File: eng/Publishing.props
Comment (wtgodbe) [inline eng/Publishing.props]: Will Arcade try to publish this to MSDL/SymServ?

**Review by pranavkm — APPROVED** [review]
If you think this is the right fix. I'm not entirely sure why pdbs are missing though.

File: eng/Publishing.props
Comment (dougbu) [reply eng/Publishing.props]: Nope; that post-build step looks for *.pdb in the downloaded PDBArtifacts folder.

File: eng/Publishing.props
Comment (dougbu) [reply eng/Publishing.props]: The root cause is not building the Microsoft.Web.Xdt.Extensions project. That's the only project we (a) need symbols for (to avoid an incomplete set of symbols) and (b) don't package (because the DLL ships in another package).

**Comment by dougbu** [comment]
Internal build: https://dev.azure.com/dnceng/internal/_build/results?buildId=558785&view=results

**Comment by dougbu** [comment]
<https://dev.azure.com/dnceng/internal/_build/results?buildId=558785&view=artifacts&type=publishedArtifacts> shows build created the PDBArtifacts artifact.

I forgot to create a default channel for my branch to ensure the missing artifact was the only problem causing the "issues" in our official builds. Tried again and also asked @JohnTortugo to execute `darc add-build-to-channel --id 45804 --channel 'General Testing'` for me, providing two chances to see symbol publication "succeed" (do nothing). Thanks @JohnTortugo❕

The real build (<https://dev.azure.com/dnceng/internal/_build/results?buildId=558921>) ran fine but still didn't try to push anything into the 'General Testing' feeds. Not sure why but maybe that channel doesn't exist in the older version of Arcade used in this branch (it's 143 commits behind 'blazor-wasm' and I built the branch directly instead of creating an internal PR)?

The promotion build <https://dnceng.visualstudio.com/internal/_build/results?buildId=558940> succeeded without issues and definitely found the PDBArtifacts artifact (see https://dnceng.visualstudio.com/internal/_build/results?buildId=558940&view=logs&j=65907e9e-dc2e-508d-4ed4-c5ec62867dbe&t=27af53db-dc59-5d1e-3865-33df1e36904b&l=177). So, we're good😺

**Comment by dougbu** [comment]
Will merge this once I prepare no-op merge PRs for 'release/3.1' and 'master' branches.

**Comment by dougbu** [comment]
:up::date: to move my `git merge-base` back to include a 'release/3.1' common ancestor. No functional changes included.

This update avoids marking ~65 'blazor-wasm' commits as being present in the 'release/3.1' branch.

**Pull request merged**
